src/main/java/com/osiris/autoplug/client/Main.java

```diff
@@ -222,9 +230,14 @@ public static void main(String[] _args) {
             utilsConfig.checkForDeprecatedSections(sharedFilesConfig);
             allModules.addAll(sharedFilesConfig.getAllInEdit());
 
+            SSHConfig sshConfig = new SSHConfig();
+            utilsConfig.checkForDeprecatedSections(sshConfig);
+            allModules.addAll(sshConfig.getAllInEdit());
+
```

Comment:
See SharedFilesConfig line 29, for how to implement live config changes,
thus its also better to have start/stop functions in your config